### ISSUE TITLE
Eliminate almost the same i18n source strings

### ISSUE THREAD

**Issue by webknjaz** [pypa/warehouse#6826]
## Bad source in Weblate

If you look at this query
https://hosted.weblate.org/translate/pypa/warehouse/uk/?q=Set+your+password+to+the+token+value%2C+including+the&type=all&offset=1
you'll notice the same string with one small difference: one has `token` variable interpolated and the other has "pypi-" hardcoded there.

So if this string was templated in both places, translators would have less work.

Another issue is that sometimes strings repeat in different places but are also included as items of translation blocks that have lists where those separate strings are also embedded. I feel like templating individual list items would simplify the situation too and some translations would be reused then.

How can we improve this? Any ideas on tracking such places in the source code?

**Comment by di** [comment]
Maybe one thing we could do here is exclude some non-translatable content (newlines? HTML tags?) from translatable strings.

Regarding interpolation vs hard-coding, that's probably just a bug that should be fixed in the corresponding template. I'm not sure how we could catch those without feedback from the translators.

**Comment by webknjaz** [comment]
Agreed, meaning that there should be a contribution path for the translators to report this. Specifically, they should somehow learn about the possibility and desire to fix such strings if they notice any.
